client/connect-account-page/index.tsx

```diff
@@ -30,13 +31,41 @@ import strings from './strings';
 import './style.scss';
 import InlineNotice from 'components/inline-notice';
 import { WooPaymentMethodsLogos } from 'components/payment-method-logos';
+import WooPaymentsLogo from 'assets/images/logo.svg?asset';
+import { __ } from '@wordpress/i18n';
+
+interface AccountData {
+	status: string;
+}
 
 const SandboxModeNotice = () => (
 	<BannerNotice icon status="warning" isDismissible={ false }>
 		{ strings.sandboxModeNotice }
 	</BannerNotice>
 );
 
+const TestDriveLoader: React.FunctionComponent< {
+	progress: number;
+} > = ( { progress } ) => (
+	<Loader className="connect-account-page__preloader">
+		<img src={ WooPaymentsLogo } alt="" />
+		<Loader.Layout>
+			<Loader.Title>
+				{ __(
+					'Creating your sandbox account',
```

Comment:
If we have strings file for this page we could use it, but if not then it's fine here

Reply:
IMO, there's not much reuse potential for these strings. We could leave them as-is.

Reply:
I agree with Anurag. I don’t believe these will be reused elsewhere.